m2c/asm_file.py

```diff
@@ -548,6 +554,14 @@ def process_label(label: str, *, kind: LabelKind) -> None:
                         data = parse_incbin(args, options, warnings)
                         if data is not None:
                             asm_file.new_data_bytes(data)
+                    elif directive == ".obj":  # decomp-toolkit label format
+                        parts = line.split()
+                        if len(parts) >= 3:
+                            try:
+                                kind = LabelKind[parts[2].upper()]
```

Comment:
LabelKind[...] exposes internal details in an ugly way and isn't searchable. Can you use some manual if-else switch instead?

Reply:
Is it fine if I set it to global by default and change it to local if the string equals local?
I did some testing and my variables get marked as static not depending on the LabelKind at all, is that normal?

Reply:
Sure, that's fine. A global label here means one that starts a function; for data it doesn't make a difference whether the label is global or not. And yeah I think m2c will happily mark things as static regardless of this, it's basing that decision on the context file instead.

Reply:
I see.